nncf/tensorflow/sparsity/base_algorithm.py

```diff
@@ -48,5 +49,12 @@ def __init__(self, target_model, op_names):
         super().__init__(target_model)
         self._op_names = op_names
 
-    def strip_model(self, model):
+    def strip_model(self, model) -> tf.keras.Model:
         return strip_model_from_masks(model, self._op_names)
+
+    def prepare_for_inference(self, make_model_copy: bool = True) -> tf.keras.Model:
```

Comment:
Idea for another PR, remove `strip_model` method and use `prepare_for_inference` instead.
There is need inspect export pipeline in Torch and TF, may be need refactor export.